src/CftfBundle/Entity/LsItem.php

```diff
@@ -948,14 +948,17 @@ function (LsItem $item) {
 
     /*
      * Find all children items of selected item
+     * @return array|null
```

Comment:
It does not look like this method could ever return `null`, so should be removed from the annotation.